### PR TITLE
13553 Support PHP 8.1

### PR DESCRIPTION
**Pull request by paul-m**

### CODE REVIEW

**Review by jastraat — COMMENTED** [review]
DDEV is still the repo; per our discussion, we're removing it!

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# This is the top-most .editorconfig file; do not search in parent directories.
```
Comment (jastraat) [inline .editorconfig]: I haven't seen an .editorconfig file in the other repos. Should there be one here?

**Comment by paul-m** [comment]
Still need to follow-up with https://github.com/GetDKAN/mock-chain/issues/15

**Review by jastraat — APPROVED** [review]
All matrix automated tests for PHP 7.4, 8.0, and 8.1 passing (as well as with prefer lowest setting). Codesniffer with PSR1 and PSR2 rules. Codeclimate added. DDEV files removed. Lots of explicit typing of variables. LGTM!

**Pull request merged**